util/syspolicy/handler.go

```diff
@@ -6,11 +6,12 @@ package syspolicy
 import (
 	"errors"
 	"sync/atomic"
+	"testing"
 )
 
 var (
-	handlerUsed atomic.Bool
-	handler     Handler = defaultHandler{}
+	policyHandlerUsed atomic.Bool
```

Comment:
switch back to the original names? Two reasons:
1. The `policy` part is redundant considering this is the `syspolicy` package
2. Makes this PR smaller